tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -162,6 +163,10 @@ def bookmark(self, bookmark_title):
     def number_of_columns(self):
         return int(self.marionette.find_element(*self._landing_page_locator).get_attribute('cols'))
 
+    @property
+    def is_homescreen_visible(self):
```

Comment:
I deleted an earlier comment. I'm not sure what Johan is referring to that you fixed. However, since this method does a wait, it should be wait_for_homescreen_visible(). The is_ type methods do a single immediate check.